models/DataObject/Localizedfield/Dao.php

```diff
@@ -702,7 +702,7 @@ public function createLocalizedViews(): void
 
             // create query
             $sql = sprintf(
-                'IF(`%s`.`%s` IS NULL OR STRCMP(`%s`.`%s`, "") = 0, %s, `%s`.`%s`)',
+                'CONVERT(IF(`%s`.`%s` IS NULL OR STRCMP(`%s`.`%s`, "") = 0, %s, `%s`.`%s`) USING utf8)',
```

Comment:
Haven't fully reproduced yet, but having this `CONVERT` function might be quite resource expensive/perfomance killer, should be in a migration and/or some collation change instead, right?